packages/app-desktop/gui/Sidebar/Sidebar.tsx

```diff
@@ -20,6 +20,7 @@ import Logger from '@joplin/lib/Logger';
 import { FolderEntity } from '@joplin/lib/services/database/types';
 import stateToWhenClauseContext from '../../services/commands/stateToWhenClauseContext';
 import { store } from '@joplin/lib/reducer';
+import { getFolderUrl, getTagUrl } from '@joplin/lib/ProtocolUtils';
```

Comment:
I think all these callback URL functions should either be in their own namespace so that you can have for example `callbackUrls.getFolderUrl` or else all functions should be consistently using "callbackUrl" and not just "url". So in that case it would be getFolderCallbackUrl and getTagCallbackUrl.

It's certainly a bit verbose but the nature of this feature means the code is a bit everywhere so it's good to be able to quickly find it and identify it if needed.